lessons/00-getting-ready/README.md

```diff
@@ -152,3 +149,20 @@ node_modules
 ```
 
 Which will ignore the `node_modules` folder — standard for a Node repository.
+
+##Class Overview
+
+###What will class look like?
+We'll spend the first ~five weeks of class on a series of topics which are central to web development, including JavaScript, Node and Express, databases, jQuery and Ajax, templating and CSS, APIs, clientside frameworks, and deployment and scaling.
+
+For the three or four weeks after that, you'll work on labs in pairs, then you'll spend the rest of the semester on a team project.
+
+For this semester we are trying out a couple different techniques in class:
+
+* **Lectures** will never be more than 30 minutes long, and will provide an overview of the lesson for the day.
+* **Individual Readings** in the form of READMEs will be done in class at your own pace.  You are encouraged to ask the teaching team any questions and discuss things with your classmates! The homework assignment for the next class will be at the end of these READMEs.  You are welcome to skip around the reading however you like, or start your homework in class.
```

Comment:
Yeah I was debating that, and I thought it was a bit confusing with a separate homework section because I want to emphasize that the readings and the homework are like one document, and they can skip around however they like, or even start the homework without reading and just reference the reading when they get stuck.